src/entities/SecurityToken/SecurityToken.ts

```diff
@@ -13,28 +13,74 @@ import { Controller } from './Controller';
 import { PolymathError } from '../../PolymathError';
 import { ErrorCode, Version } from '../../types';
 
+/**
+ * Represents a unique security token
+ */
 export interface UniqueIdentifiers {
+  /**
+   * symbol of the security token
+   */
   symbol: string;
 }
 
+/**
+ * Check if the provided value is of type [[UniqueIdentifiers]]
+ *
+ * @param identifiers - internal security token representation
+ */
 function isUniqueIdentifiers(identifiers: any): identifiers is UniqueIdentifiers {
   const { symbol } = identifiers;
 
   return typeof symbol === 'string';
 }
 
+/**
+ * Represents a single Security Token
+ */
 export interface Params {
+  /**
+   * name of the Security token
+   */
   name: string;
+  /**
+   * address of the Security token
+   */
   address: string;
+  /**
+   * the owner of the Security Token
+   */
   owner: string;
+  /**
+   * the off-chain data associated with the Security Token
+   */
   tokenDetails: string;
+  /**
+   * module version of the security token
+   */
   version: Version;
+  /**
+   * granularity level of the token
+   */
   granularity: number;
+  /**
+   * total number of market cap to be created
+   */
   totalSupply: BigNumber;
+  /**
+   * checkpoint id used to query historical balances
+   */
   currentCheckpoint: number;
+  /**
+   * ethereum address which will holds the STs
```

Comment:
"default treasury wallet used by some features. For example, if an STO reaches its end date (or is finalized before that), remaining unsold tokens get transferred to this wallet unless otherwise specified by the STO itself"